docs/core/reference/corectl.md

```diff
@@ -126,13 +126,23 @@ corectl create-block-keypair
 
 ### `create-token`
 
-Generates a new access token with the given name. Flag -net means to create a network token,
-otherwise it will create a client token.
+Generates a new access token with the given name
+and grants it access to the named policy.
 
 ```
-corectl create-token [-net] [name]
+corectl create-token [-net] [name] [policy]
 ```
 
+If no policy is given,
```

Comment:
I was trying to do http://rhodesmill.org/brandon/2012/one-sentence-per-line/ but I won't insist.